lib/charms/tls_certificates_interface/v4/tls_certificates.py

```diff
@@ -401,13 +473,13 @@ def to_certificate_request(self) -> "CertificateRequest":
         )
 
     @staticmethod
```

Comment:
```suggestion
    @classmethod
```

Same comment on this method.

Reply:
I'll resolve this one here so that we keep the discussion in 1 place.